diff --git a/src/oph_term.c b/src/oph_term.c
index 7fc199c..b318d15 100644
--- a/src/oph_term.c
+++ b/src/oph_term.c
@@ -1769,7 +1769,7 @@ int main(int argc, char **argv, char **envp) {
 
     //Auto-completion
     if (!exec_statement) {
-    	rl_attempted_completion_function = (CPPFunction *)oph_term_completion;
+    	rl_attempted_completion_function = (rl_completion_func_t *)oph_term_completion;
     }
 
     /* MAIN LOOP */
