diff -ru /src/drivers/OPH_EXPLORENC_operator.c /src/drivers/OPH_EXPLORENC_operator.c
--- a/src/drivers/OPH_EXPLORENC_operator.c	2016-04-05 16:56:22.013763890 +0000
+++ b/src/drivers/OPH_EXPLORENC_operator.c	2016-04-05 16:40:04.519051679 +0000
@@ -401,15 +401,15 @@
 
   //1 - Set up struct to empty values
   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->nc_file_path = NULL;
-  ((NETCDF_var)(   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->measure   )).dims_name = NULL;
-  ((NETCDF_var)(   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->measure   )).dims_id = NULL;
-  ((NETCDF_var)(   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->measure   )).dims_unlim = NULL;
-  ((NETCDF_var)(   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->measure   )).dims_length = NULL;
-  ((NETCDF_var)(   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->measure   )).dims_type = NULL;
-  ((NETCDF_var)(   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->measure   )).dims_oph_level = NULL;
-  ((NETCDF_var)(   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->measure   )).dims_start_index = NULL;
-  ((NETCDF_var)(   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->measure   )).dims_end_index = NULL;
-  ((NETCDF_var)(   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->measure   )).dims_order = NULL;
+//  ((NETCDF_var)(   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->measure   )).dims_name = NULL;
+//  ((NETCDF_var)(   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->measure   )).dims_id = NULL;
+//  ((NETCDF_var)(   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->measure   )).dims_unlim = NULL;
+//  ((NETCDF_var)(   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->measure   )).dims_length = NULL;
+//  ((NETCDF_var)(   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->measure   )).dims_type = NULL;
+//  ((NETCDF_var)(   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->measure   )).dims_oph_level = NULL;
+//  ((NETCDF_var)(   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->measure   )).dims_start_index = NULL;
+//  ((NETCDF_var)(   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->measure   )).dims_end_index = NULL;
+//  ((NETCDF_var)(   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->measure   )).dims_order = NULL;
   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->limit = 0;
   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->show_id = 0;
   ((OPH_EXPLORENC_operator_handle*)handle->operator_handle)->show_index = 0;
diff -ru /src/drivers/OPH_IMPORTNC2_operator.c /src/drivers/OPH_IMPORTNC2_operator.c
--- a/src/drivers/OPH_IMPORTNC2_operator.c	2016-04-05 16:56:22.009763968 +0000
+++ b/src/drivers/OPH_IMPORTNC2_operator.c	2016-04-05 16:42:58.967589428 +0000
@@ -76,15 +76,15 @@
   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->compressed = 0;
   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->import_metadata = 0;
   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->check_compliance = 0;
-  ((NETCDF_var)(   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->measure   )).dims_name = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->measure   )).dims_id = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->measure   )).dims_unlim = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->measure   )).dims_length = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->measure   )).dims_type = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->measure   )).dims_oph_level = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->measure   )).dims_start_index = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->measure   )).dims_end_index = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->measure   )).dims_concept_level = NULL;
+  // ((NETCDF_var)(   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->measure   )).dims_name = NULL;
+  // ((NETCDF_var)(   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->measure   )).dims_id = NULL;
+  // ((NETCDF_var)(   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->measure   )).dims_unlim = NULL;
+  // ((NETCDF_var)(   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->measure   )).dims_length = NULL;
+  // ((NETCDF_var)(   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->measure   )).dims_type = NULL;
+  // ((NETCDF_var)(   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->measure   )).dims_oph_level = NULL;
+  // ((NETCDF_var)(   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->measure   )).dims_start_index = NULL;
+  // ((NETCDF_var)(   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->measure   )).dims_end_index = NULL;
+  // ((NETCDF_var)(   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->measure   )).dims_concept_level = NULL;
   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->cwd = NULL;
   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->user = NULL;
   ((OPH_IMPORTNC2_operator_handle*)handle->operator_handle)->run = 1;
diff -ru /src/drivers/OPH_IMPORTNC3_operator.c /src/drivers/OPH_IMPORTNC3_operator.c
--- a/src/drivers/OPH_IMPORTNC3_operator.c	2016-04-05 16:56:22.016763831 +0000
+++ b/src/drivers/OPH_IMPORTNC3_operator.c	2016-04-05 16:43:31.424945216 +0000
@@ -76,15 +76,15 @@
   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->compressed = 0;
   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->import_metadata = 0;
   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->check_compliance = 0;
-  ((NETCDF_var)(   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->measure   )).dims_name = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->measure   )).dims_id = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->measure   )).dims_unlim = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->measure   )).dims_length = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->measure   )).dims_type = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->measure   )).dims_oph_level = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->measure   )).dims_start_index = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->measure   )).dims_end_index = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->measure   )).dims_concept_level = NULL;
+  /// ((NETCDF_var)(   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->measure   )).dims_name = NULL;
+  /// ((NETCDF_var)(   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->measure   )).dims_id = NULL;
+  /// ((NETCDF_var)(   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->measure   )).dims_unlim = NULL;
+  /// ((NETCDF_var)(   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->measure   )).dims_length = NULL;
+  /// ((NETCDF_var)(   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->measure   )).dims_type = NULL;
+  /// ((NETCDF_var)(   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->measure   )).dims_oph_level = NULL;
+  /// ((NETCDF_var)(   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->measure   )).dims_start_index = NULL;
+  /// ((NETCDF_var)(   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->measure   )).dims_end_index = NULL;
+  /// ((NETCDF_var)(   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->measure   )).dims_concept_level = NULL;
   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->cwd = NULL;
   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->user = NULL;
   ((OPH_IMPORTNC3_operator_handle*)handle->operator_handle)->run = 1;
diff -ru /src/drivers/OPH_IMPORTNC_operator.c /src/drivers/OPH_IMPORTNC_operator.c
--- a/src/drivers/OPH_IMPORTNC_operator.c	2016-04-05 16:56:22.014763870 +0000
+++ b/src/drivers/OPH_IMPORTNC_operator.c	2016-04-05 16:42:03.393692429 +0000
@@ -76,15 +76,15 @@
   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->compressed = 0;
   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->import_metadata = 0;
   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->check_compliance = 0;
-  ((NETCDF_var)(   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->measure   )).dims_name = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->measure   )).dims_id = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->measure   )).dims_unlim = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->measure   )).dims_length = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->measure   )).dims_type = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->measure   )).dims_oph_level = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->measure   )).dims_start_index = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->measure   )).dims_end_index = NULL;
-  ((NETCDF_var)(   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->measure   )).dims_concept_level = NULL;
+  // ((NETCDF_var)(   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->measure   )).dims_name = NULL;
+  // ((NETCDF_var)(   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->measure   )).dims_id = NULL;
+  // ((NETCDF_var)(   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->measure   )).dims_unlim = NULL;
+  // ((NETCDF_var)(   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->measure   )).dims_length = NULL;
+  // ((NETCDF_var)(   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->measure   )).dims_type = NULL;
+  // ((NETCDF_var)(   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->measure   )).dims_oph_level = NULL;
+  // ((NETCDF_var)(   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->measure   )).dims_start_index = NULL;
+  // ((NETCDF_var)(   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->measure   )).dims_end_index = NULL;
+  // ((NETCDF_var)(   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->measure   )).dims_concept_level = NULL;
   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->cwd = NULL;
   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->user = NULL;
   ((OPH_IMPORTNC_operator_handle*)handle->operator_handle)->run = 1;
